code/modules/reagents/chemistry/reagents/trickwine_reagents.dm

```diff
@@ -42,14 +38,16 @@
 	description = "A specialized brew utilized by members of the Saint-Roumain Militia, designed to assist in temperature regulation while working in hot environments. Known to give one the cold shoulder when thrown."
 	color = "#C0F1EE"
 	boozepwr = 70
-	taste_description = "a cold night on the hunt"
+	taste_description = "a weighty meat, undercut by a mild pepper."
 	glass_name = "Icewine"
 	glass_desc = "A specialized brew utilized by members of the Saint-Roumain Militia, designed to assist in temperature regulation while working in hot environments. Known to give one the cold shoulder when thrown."
 	breakaway_flask_icon_state = "baflaskicewine"
 
 /datum/reagent/consumable/ethanol/trickwine/ice_wine/on_mob_life(mob/living/M)
 	M.adjust_bodytemperature(-5 * TEMPERATURE_DAMAGE_COEFFICIENT, M.get_body_temp_normal())
-	M.adjustFireLoss(-1)
+	if(prob(10))
+		to_chat(M, span_notice("Sweat runs down your body."))
+		M.adjustFireLoss(-0.5)
```

Comment:
Very strong nerf. Whats your reasoning?

Reply:
I wanted to move it away from being on par with dedicated healing chemicals, while retaining the use it had